components/nft-sale-widget/commons/sale-type-switch.tsx

```diff
@@ -24,13 +25,25 @@ export const SaleTypeSwitch = ({
           }
         }}
       />
+
       <Text
         fontFamily={"Inter Medium"}
         fontSize={{ base: 12, md: 14 }}
         color={"#585858"}
       >
         Original Sale
       </Text>
+      <Tooltip
+        label="Toggle to view available copies"
```

Comment:
what if we are already seeing copies?

Reply:
Maybe this message should change depending on toggle state

Reply:
Great idea! I will implement